Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsPropertyConversionService.cs

```diff
@@ -38,5 +50,19 @@ private TimeSpan ToTimeSpan(string value)
                 _ => throw new ArgumentNullException()
             };
         }
+
+        private ulong ParseToUserId(string value)
+        {
+            var id = ulong.Parse(_exMention.Match(value).Value);
+            var user = Server.GetUser(id).Result;
```

Comment:
dlaczaego za każdym razem pobierasz użytkownika z serwera?
to tylko dodatkowe obciążenie

plus pomyśl jeszcze o obsłudze kogoś kto wyszedł z serwera
przykładowo chcielibyśmy sprawdzić historie warnów użytkownika, który dostał bana

Reply:
ja z @gagyn uzgadniałem to czy sprawdzić istnienie użytkownika w trakcie obsługi komendy czy w trakcie parsowania to miałoby w parsowaniu, ale skoro tak mówisz to po prostu przeniosę to do komend tak jak było to wcześniej.

Reply:
mamy takie komendy jak -messages albo (niedługo dojdzie) -warns
i chcielibyśmy mieć możliwość obsługi użytkowników, których już z nami nie ma

to 2 przykłady, ale możliwe że w przyszłości będzie tego więcej